### PR TITLE
electronic filed incorporation

### PR DESCRIPTION
**Pull request by danfalc**
Before creating this pull request, I confirm that;

- [x] I have adhered to the [Pull Request Review Checklist](https://companieshouse.atlassian.net/wiki/spaces/TT/pages/27624876/Pull+Request+Review+Checklist)
- [x] I have read the [Automation Best Practices](https://companieshouse.atlassian.net/wiki/spaces/TT/pages/215416988/Automation+Best+Practises)
- [x] Checkstyle passes locally

### CODE REVIEW

File: src/test/java/uk/gov/companieshouse/data/datamodel/Director.java [hunk 1]
```diff
@@ -30,6 +30,7 @@ public class Director {
     /**
      * Create a new director with default values.
      */
+
```
Comment (rsuller) [inline src/test/java/uk/gov/companieshouse/data/datamodel/Director.java:33]: Unattached comment?

File: src/test/java/uk/gov/companieshouse/stepdefinitions/CommonStepDefs.java [hunk 2]
```diff
@@ -98,7 +98,7 @@ public void companyHistoryInformationIsUpdated(String formName) {
         Form form = Form.getFormByType(formName);
         String descriptionToCheck;
         companySearchPage
-                .findCompanyByNumberFromMenu()
+                .findCompanyByNameFromMenu()
```
Comment (rsuller) [inline src/test/java/uk/gov/companieshouse/stepdefinitions/CommonStepDefs.java:101]: Why is no name being passed as an argument? iS there a hardcoding of the company name in this method? Yes I am being too lazy to check 😬

File: src/test/resources/uk/gov/companieshouse/features/incorporation/in01.feature [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  * For the use of people who want to conduct a business or other activity for community benefit
+  * CICs use the standard IN01 form
+  * Must select the CIC indicator check box
+  * Must provide a CIC name ending e.g. ‘community interest company’, ‘CIC’ or the Welsh equivalent
+  * Must select bespoke articles of incorporation
+
+  @electronic
+  Scenario: Electronic Filed auto accepted IN01 form successfully processing a new company limited by shares
+    Given I am logged in as a user in the "EF New Companies" organisational unit  
+    And I process an e-filed "IN01" form for a private limited company
```
Comment (rsuller) [inline src/test/resources/uk/gov/companieshouse/features/incorporation/in01.feature:47]: Change this 'And' to a 'When' as it is the testable action in this scenario.

File: src/test/java/uk/gov/companieshouse/stepdefinitions/CommonStepDefs.java (hunk 2 above)
Comment (danfalc) [reply src/test/java/uk/gov/companieshouse/stepdefinitions/CommonStepDefs.java:101]: This gets the company name from what is stored in memory

File: src/test/resources/uk/gov/companieshouse/features/incorporation/in01.feature (hunk 3 above)
Comment (danfalc) [reply src/test/resources/uk/gov/companieshouse/features/incorporation/in01.feature:47]: Changed

File: src/test/java/uk/gov/companieshouse/data/datamodel/Director.java (hunk 1 above)
Comment (danfalc) [reply src/test/java/uk/gov/companieshouse/data/datamodel/Director.java:33]: Changed

**Review by rsuller — APPROVED** [review]

**Pull request merged**